### DISCUSSION TITLE
Socket IO Official Website Documentation Bug?

### DISCUSSION THREAD

**Discussion by dragonlobster** [socketio/socket.io#4324] [Q&A] [answered]
hello there seems to be a small problem, i'm on chrome and when I go on the socket io official website documentation: https://socket.io/docs/v4/server-api , on the right whenever i press one of the links on the right side (for example `sockettoroom`), the link in address bar changes correctly to `/#sockettoroom`, but the website doesn't direct me there. Just a small annoyance, in fact if I try to go to https://socket.io/docs/v4/server-api/#sockettoroom, it just takes me to the top.

**Comment by darrachequesne** [comment, marked as answer]
Fixed by https://github.com/socketio/socket.io-website/pull/334.
